search_gov_crawler/benchmark.py

```
@@ -39,6 +40,8 @@
 from search_gov_crawler.search_gov_spiders.extensions.json_logging import LOG_FMT
 from search_gov_crawler.search_gov_spiders.utility_files.crawl_sites import CrawlSites
 
+load_dotenv()
```

Did you add these just for east of development?  Or were you thinking we should start using .env files in higher environments?

If we are officially adopt `dotenv` I think we should update the main readme with instructions on how to use since `.env` is already excluded by our main .gitigore.

---

@selfdanielj Yeah, this is mostly for dev/internal purposes. This makes it so we don't have to always manually set our env variables when testing. My `.env` file looks like this:
```
SPIDER_ES_INDEX_NAME="i14y-documents-spider"
SPIDER_ES_INDEX_ALIAS="i14y-documents"
ES_HOSTS="http://localhost:9200"
ES_USER="username"
ES_PASSWORD="password"
SPIDER_INDEX_TO_ELASTICSEARCH="true"
SPIDER_URLS_API="https://fake-api.com/urls"
```

---

It does not effect anything production wise, since it'll just silently fail if the `.env` file does not exist

---

sure, i think it would be helpful to call this out in the readme saying we support .env files and where to place them